app/modules/asset_groups/schemas.py

```diff
@@ -180,9 +191,6 @@ class AssetGroupSightingAsSightingSchema(ModelSchema):
     timeSpecificity = base_fields.Function(
         AssetGroupSighting.config_field_getter('timeSpecificity')
     )
-    locationId = base_fields.Function(
```

Comment:
This is not the removal of a field from the API, this is the removal of duplication, see line 204 below